cmd/app/main-win.go

```diff
@@ -0,0 +1,46 @@
+//go:build windows
+// +build windows
+
+package app
+
+import (
+	"os"
+	"syscall"
+
+	"github.com/spring-financial-group/jx-semanticcheck/pkg/cmd"
```

Comment:
Remember to remove references to spring-financial-group

Reply:
eek - yep I'll fix that